content/20.treatments.md

```diff
@@ -11,48 +11,91 @@ In the following sections, we critically appraise the literature surrounding the
+Perhaps, the TCZ treatment would best suit patients with severely compromised lungs due to the Covid-19 infection and are therefore at greater risk of death, in order to stop the uncontrolled immune response before it’s too late. 
 
 ##### Summary
 
 Summarize the state of the symptom management approach.
 
 ### Small Molecule Drugs for COVID-19
 
-Antivirals are an emerging category of drugs.
-Unlike antibiotics, they do not kill viruses.
-Rather, they inhibit the proliferation of a virus.
-Categories may include therapies that inhibit viral proteins, inhibit viral entry, and more
-Antivirals have been used to treat XYZ diseases through [what mechanisms or approaches are used?]
+The replication cycle of a virus within an epithelial host cell includes 6 basic steps which can be shortly summarised as follow: i) attachment of the virus to the host cell; ii) penetration by endocytosis; iii) uncoating, classically defined as the release of viral contents into the host cell; iv) biosynthesis, during which the viral genetic material enters the nucleus where it gets replicated; v) assembly, where viral proteins are translated and new viral particles are assembled; vi) release, when the new viruses are released into the extracellular environment [@isbn:978-1405136457]. 
+Antiviral drugs do not kill the virus, rather they inhibit its amplification by impairing one of these steps. 
+Nowadays, many of these drugs act during the biosynthesis step in order to inhibit the virus’ genetic material replication. Importantly, Corona viruses, including Covid-19, are RNA viruses. 
+Differently from a DNA virus, which can use the host enzymes to propagate itself, RNA viruses depends on their own polymerase, the RNA-dependent RNA polymerase (RdRP), in order to be replicated [@doi:10.1007/978-1-4939-2438-7; @doi:10.1002/jmv.25761].
 
-Add a subcategory (using ####) for each category of antiviral treatment
+#### Nucleotide Analogs
 
-#### Viral Protein Targeting Drugs
+Why one might use nucleotide analogs.
 
-Brief background on the therapeutic.
+##### Avigan
 
-##### Nucleoside Analogues
+Avigan (Flavipiravir, from Toyama chemical Fujifilm) is a drug which has been found effective to block viral amplification in the Influenza virus infection. 
+Specifically, Avigan is a nucleoside precursor efficiently recognised as guanosine and adenosine analogue by the virus polymerase (RNA-dependent RNA polymerase). 
+While a single incorporation does not influence RNA transcription, multiple events of incorporation lead to the arrest of RNA synthesis [@doi:10.1371/journal.pone.0068347].
+Importantly It was already shown that Avigan is able to give 100% coverage against the Ebola virus in mice [@doi:10.1038/s41422-020-0282-0]. 
+Furthermore, there are evidences that drug might also work against Corona virus infection. 
+For instance, a recent study showed its effectiveness compared to other antivirals such as lopinavir and ritonavir. 
```

Comment:
Ref?

Reply:
Cgreene sorry I was working on another review for my lab. I will now take a look at all your suggestions

Reply:
I also wanted to add more about the mechanism of action of tocilizumab, to be more specific. Will work on it right now

Reply:
I will add now all the references

Reply:
This is the ref you were asking about https://www.sciencedirect.com/science/article/pii/S2095809920300631

Reply:
@cgreene  I accepted all your suggestions and added something more on the mechanism of action of tocilizumab starting from line 40. please let me know whether you think I am going to the right direction with it.